java/core/src/test/org/apache/orc/TestVectorOrcFile.java

```diff
@@ -538,7 +538,7 @@ public void testStringAndBinaryStatistics(Version fileFormat) throws Exception {
 
     assertEquals(3, stats[1].getNumberOfValues());
     assertEquals(15, ((BinaryColumnStatistics) stats[1]).getSum());
-    assertEquals("count: 3 hasNull: true bytesOnDisk: 28 sum: 15", stats[1].toString());
+    assertEquals("count: 3 hasNull: true bytesOnDisk: 30 sum: 15", stats[1].toString());
```

Comment:
It seems that after enabling ZSTD by default, the size becomes larger, right?

Reply:
It's a noise because the test file is too tiny, @deshanxiao .

Reply:
For general data like the following, `zstd` is smaller than `gzip`.
```
$ java -jar core/target/orc-benchmarks-core-*-uber.jar generate data -f orc -d sales -s 1000000
$ ls -alh data/generated/sales
total 721968
drwxr-xr-x  5 dongjoon  staff   160B Jan  8 17:27 .
drwxr-xr-x  3 dongjoon  staff    96B Jan  8 17:27 ..
-rw-r--r--  1 dongjoon  staff   102M Jan  8 17:27 orc.gz
-rw-r--r--  1 dongjoon  staff   115M Jan  8 17:27 orc.snappy
-rw-r--r--  1 dongjoon  staff   101M Jan  8 17:27 orc.zstd
```